src/components/Item.js

```diff
@@ -24,8 +24,28 @@ const Item = ({ item, handleChange }) => {
     setChecked(check);
   }, [item]);
 
+  const addNextPurchaseStyling = item => {
+    let color = '';
+
+    if (item.status === 'inactive') {
+      color = '#747d8c';
+    } else {
+      if (item.nextPurchase <= 7) {
+        color = '#eb2f06';
+      } else if (item.nextPurchase > 7 && item.nextPurchase < 30) {
+        color = '#eccc68';
+      } else if (item.nextPurchase >= 30) {
+        color = '#2ed573';
+      }
+    }
+    return color;
+  };
+
   return (
-    <li>
+    <li
+      className="list-item"
```

Comment:
If you take my suggestion of assigning a class name, you'd append it here, so something like:

    className={ `list-item ${addNextPurchaseStyling(item)}` }